src/ObjectGraph.es6.js

```diff
@@ -224,7 +250,9 @@ ObjectGraph.prototype.visitPropertyDescriptors = function(o, metadataMap) {
 // Visit an object, o. Return an id for the object, which may contain type
 // information (e.g., number, boolean, null), or indicate the unique identity
 // of the object itself.
-ObjectGraph.prototype.visitObject = function(o) {
+ObjectGraph.prototype.visitObject = function(o, opt) {
```

Comment:
We control all call-sites to `visitObject`, right? (I.e., it's an internal API). Could we just update call sites and always pass `opt`? Also, what all is stored in `opt`? Can we give it a more meaningful name?